examples/cloudflare/src/index.ts

```diff
@@ -1,33 +1,44 @@
-import { connect, Connection, ExecutedQuery } from '@planetscale/database'
+import { Client, Connection, ExecutedQuery, connect } from '@planetscale/database'
 
 export interface Env {
-  PSCALE_HOST: string
-  PSCALE_USERNAME: string
-  PSCALE_PASSWORD: string
+  DATABASE_HOST: string
+  DATABASE_USERNAME: string
+  DATABASE_PASSWORD: string
+  REPLICA_DATABASE_HOST: string
+  REPLICA_DATABASE_USERNAME: string
+  REPLICA_DATABASE_PASSWORD: string
 }
 
-const CURRENT_YEAR = 2023
+const CURRENT_YEAR = 2024
```

Comment:
Rather than having to update this variable each year, what if we just dynamically fetched it? 👀 

```suggestion
const CURRENT_YEAR = new Date().getFullYear()
```

Reply:
I'm not sure if a season spans calendar years, and we don't auto-write the new year

Reply:
Ahhhhhhh, got it, got it. Okay!